Makefile

```diff
@@ -73,7 +73,7 @@ shell: ## Exec into a container with the kubicorn source mounted inside
 	--rm ${SHELL_IMAGE} /bin/bash
 
 lint: install-tools ## check for style mistakes all Go files using golint
-	golint $(PKGS)
+	~/go/bin/golint $(PKGS)
```

Comment:
Argh. This shouldn't have happened. Sorry for this mess, been in a hurry to get the family into vacation and this PR merged 👍